app/publish-update-retire-content/standard-content-types/consultations.md

```diff
@@ -0,0 +1,194 @@
+
+Update the consultation page to explain the delay if it takes longer than that. Give an estimate about when the response will be ready if you can.
+
+Do not add any more information or attachments to the page after the government response has been published.
+
+## Create a draft
+
+{% include 'reusable-content/signon-account-whitehall.md' %}
+
+{% include 'reusable-content/remove-history-mode.md' %}
+
+### If you're creating a new consultation
+
+1. Go to [Whitehall Publisher](https://whitehall-admin.publishing.service.gov.uk/government/admin/).
+2. Select the 'New document' tab.
+3. Select 'Consultation' and then select the 'Next' button.
+4. If the consultation is not in English, tick 'Create a foreign language only consultation' and select the language.
+5. Add the title and summary.
+6. In the body, add a brief account of what you are asking people about and who the consultation is aimed at. Keep it to under 100 words, and do not include the closing date as it's added automatically.
+7. Add an opening and closing date for the consultation. You can also add a specific opening and closing time.
+8. If the consultation documents and the ways of responding are not hosted on GOV.UK, tick 'This consultation is held on another website' and include the link.
+9. Add details about the 'Ways to respond'. This could be a link to an online form, an email address, a postal address or a downloadable form. If you're attaching a downloadable form, make sure you've correctly [formatted the attachment](/formatting-content/attachments/).
+10. Tick the UK nations where the content applies under 'Excluded nations (required)'. If your content does not apply to a nation, you can optionally provide a link to alternative content for users in that nation.
+11. Tick the box to confirm you've considered the [consultation principles](https://www.gov.uk/government/publications/consultation-principles-guidance).
+12. Select 'Limit access' under 'Settings' if your draft is sensitive. This means only users from organisations tagged to the content can see it before it's published. It will not be copied across to [Integration](https://signon.integration.publishing.service.gov.uk/).
+13. Select the 'Save' button at the bottom of the page.
+
+{% include 'reusable-content/style-formatting-extra.md' %}
+
+Once that's done, you can add the consultation documents as attachments.
+
+Go to the 'Attachments' tab. Your attachments can either be file attachments or HTML attachments.
+
+If you're uploading a file attachment:
+
+- make sure you've correctly [formatted the attachment](/formatting-content/attachments/) before you upload it
+- do not tick 'Attachment is accessible' – this will let users see a box with information about requesting the attachment in an accessible format, like audio or Braille
+
+If you're uploading a HTML attachment, read the [formatting guidance](/formatting-content/) for help.
```

Comment:
Could this link directly to the relevant guidance, or is this more about formatting content in an HTML attachment?